contracts/oraclize/TestableMarketContractFactoryOraclize.sol

```diff
@@ -78,4 +83,13 @@ contract TestableMarketContractFactoryOraclize is Ownable {
         require(registryAddress != address(0));
         marketContractRegistry = registryAddress;
     }
+
+    /*
+    currently adding this function pushes us over the edge for gas, for the time being we can leave it out.
```

Comment:
Do we really want to switch factories anyway?

Reply:
The only reason I can see to switch factories would be in production if we wanted to fix an issue in just the CollateralPoolFactory contract, we could and then hot swap the contracts.